setup-scripts/setup-siad.sh

```diff
@@ -0,0 +1,56 @@
+#! /usr/bin/env bash
+
+set -e # exit on first error
+
+# Install Go 1.13.11
+wget -c https://dl.google.com/go/go1.13.11.linux-amd64.tar.gz
+sudo tar -C /usr/local -xzf go1.13.11.linux-amd64.tar.gz
+rm go1.13.11.linux-amd64.tar.gz
```

Comment:
Maybe we can move the go version and the sia version to variable, so it's easy to update. Otherwise there would be one `1.4.8` that we forget to change to `1.4.9`. :)

Reply:
Good idea, extracted :)